spec/message_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'message'
+
+describe 'Message' do
+  describe'.all' do
+    it 'stores the messages' do
+      connection = PG.connect(dbname: 'chitter_test')
+
+      #add the test data
+      connection.exec("INSERT INTO peeps (message) VALUES ('Today''s first peep');")
+      connection.exec("INSERT INTO peeps (message) VALUES ('Today''s second peep');")
+      connection.exec("INSERT INTO peeps (message) VALUES ('Today''s third peep');")
+
+      messages = Message.all 
+      list_of_messages = messages.map { |item| item.message }
+
+      expect(list_of_messages).to eq ["Today's first peep","Today's second peep","Today's third peep"]
```

Comment:
this is a great way of testing the messages. I wasn't sure how to do it properly in a unit test but this makes a lot of sense.